Libraries/oneMKL/guided_cuRAND_examples_SYCL_migration/02_sycl_dpct_migrated/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl -std=c++17")
-set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lmkl_sycl -lmkl_intel_ilp64 -lmkl_sequential -lmkl_core")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl -std=c++17 -qmkl")
```

Comment:
Same here, we use -qmkl only for adding oneMKL header files to the compilation line here otherwise the build works only if all requires oneMKL header files are in CPATH. Threading layer is defined on the next line